packages/docs/components/page-configs/va-carousel/api-options.ts

```diff
@@ -0,0 +1,27 @@
+import { ManualApiOptions } from 'vuestic-ui/src/services/api-docs/ManualApiOptions'
```

Comment:
Needs to be more visible: 
![image](https://user-images.githubusercontent.com/5394573/96589753-de3cae80-12ed-11eb-8d6a-badf13100cd3.png)